doc/aws-deployment.md

```diff
@@ -0,0 +1,386 @@
+    directory][1] to make the directory accessible to the `id` group.
+
+4.  Configure Nginx as a reverse-proxy (hack the
+    `/etc/nginx/sites-available/default` file see: [NGINX Reverse Proxy][2] and
+    the example site config described below.
+
+5.  The Nginx will also require SSL certificates; the current configuration of
+    the server seems to come with the Let's Encrypt certificates already in
+    place.
+
+6.  Nginx will need limit on uploads increasing to around 200 Mb in
+    `/etc/nginx/nginx.conf` to support ingesting large files (add this in the
+    `http { …` section as `client_max_body_size 200M;`
+
+## Build the Docker containers
+
+Two containers are used to deploy Mauro: one contains the DBMS the other the
+Mauro front and back ends.  The building is carried out using the
+`./scripts/update-from-github.sh` script.
+
+Given the current requirement for three slightly different deployments we are
+using long-lived git branches which maintain the different configurations these
+need.  So when creating a new deployment we suggest creating a new branch (based
+off of `develop`) and updating the files which require different values—see step
+3 below.
+
+1.  Copy the `update-from-github.sh` to the `/home/build` directory (make it
+    executable).
+
+2.  Set the `BRANCH_TAG` value in the script to identify the
+    `nhsd-datadictionary-docker` branch to use on this server—make sure the
+    branch has been pushed).
+
+3.  If you are deploying to a new server (with a new hostname) check the
+    `up.sh` (`down.sh`?) and `.env` files to ensure that they are exporting the
+    correct value for `MAURO_ENDPOINT`.
+
+4.  Run the `update-from-github.sh` script.  With all the above in place the
+    site should be available via the hostname
+
+5.  Work through the "Configuring Mauro" section of the Site setup
+    documentation.
+
+6.  Ingest a suitable branch and when that has completed create a new version
+
+\pagebreak
+# Configuring Mauro
+
+With the site running there are a number of tasks to carry out before it can
+be used.
+
+## Administrator account
+
+When you first access Mauro (MDM UI, not the Orchestrator), you can sign in
+with the standard Mauro account created during bootstrap:
+
+Username: "admin@maurodatamapper.com"
+Password: "password"
+
+After sign in, you will be forced to change this password to something more
+secure.
+
+## Site URL
+
+There is a configuration token used in Mauro, particularly for email templates,
+stating what the base URL is for this Mauro instance. This must be set
+manually.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Configuration"
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "site.url"
+5.  Under "Value", enter the base URL domain this Mauro instance uses.
+6.  Click "Add property" to save.
+
+## OpenID Connect
+
+The OpenID Connect feature needs to be switched on in Mauro.
```

Comment:
We'll also need to explain how to setup an OpenID Connect provider. Fortunately this is already documented, just include a reference to this link: https://maurodatamapper.github.io/plugins/user-guides/openid-connect/openid-connect/#24-add-providers-to-mauro